static/app/views/replays/detail/console/messageFormatter.tsx

```diff
@@ -1,22 +1,86 @@
-// webpack fallback handles this
-// eslint-disable-next-line
-import {format} from 'util';
+import {Fragment} from 'react';
+import {chromeLight, ObjectInspector} from 'react-inspector';
 
 import {AnnotatedText} from 'sentry/components/events/meta/annotatedText';
 import {getMeta} from 'sentry/components/events/meta/metaProxy';
 import {BreadcrumbTypeDefault, Crumb} from 'sentry/types/breadcrumbs';
-import isObject from 'lodash/isObject';
 import {objectIsEmpty} from 'sentry/utils';
 
 interface Props {
   breadcrumb: Extract<Crumb, BreadcrumbTypeDefault>;
 }
 
+const formatRegExp = /%[sdj%]/g;
+function isObject(arg) {
+  return typeof arg === 'object' && arg !== null;
+}
+function isNull(arg) {
+  return arg === null;
+}
+const format = function (...args) {
+  const INSPECTOR_THEME = {
+    ...chromeLight,
+    BASE_BACKGROUND_COLOR: 'none',
+    OBJECT_PREVIEW_OBJECT_MAX_PROPERTIES: 1,
+  };
```

Comment:
`import ConfigStore from 'sentry/stores/configStore';`

```suggestion
  const isDark = ConfigStore.get('theme') === 'dark';

  const INSPECTOR_THEME = {
    ...(isDark ? chromeDark : chromeLight),
    BASE_BACKGROUND_COLOR: 'none',
    OBJECT_PREVIEW_OBJECT_MAX_PROPERTIES: 1,
  };
```

Reply:
I have this refactored into a separate component